share/spice/grunt/grunt.js

```diff
@@ -0,0 +1,101 @@
+(function (env) {
+    "use strict";
+    
+    // From http://stackoverflow.com/a/34064434/1546577
+    // This is used to decode entities that is often found in the descriptions.
+    function htmlDecode(input) {
+        var doc = new DOMParser().parseFromString(input, "text/html");
+        return doc.documentElement.textContent;
+    }
+    
+    // Add a star beside the name if it's an official plugin.
+    function officialPlugin(author) {
+        if(author === 'Grunt Team') {
+            return '★';
```

Comment:
Hmm, I think this may look better on the left, or as an `altSubtitle`like "Official Plugin".

I was going to suggest the icon template, but I think it's been removed...